src/wp-includes/blocks/latest-posts.php

```diff
@@ -152,10 +152,9 @@ function render_block_core_latest_posts( $attributes ) {
 				if ( $excerpt_length <= $block_core_latest_posts_excerpt_length ) {
 					$trimmed_excerpt  = substr( $trimmed_excerpt, 0, -11 );
 					$trimmed_excerpt .= sprintf(
-						/* translators: 1: A URL to a post, 2: The static string "Read more", 3: The post title only visible to screen readers. */
-						__( '… <a href="%1$s" rel="noopener noreferrer">%2$s<span class="screen-reader-text">: %3$s</span></a>' ),
+						/* translators: The static string "Read more". %1$s: A URL to a post, %2$s: Hidden accessibility text: Post title */
```

Comment:
There's some description in here that probably shouldn't be, I think these comments should only include the description of the placeholders.
```suggestion
						/* translators: 1: URL to a post, 2: post title */
```

Reply:
Indeed!
I recommend checking the handbook for best practices like that

Reply:
I added `The static string "Read more"` here as the "Read more" string is now part of this single placeholder. Is it OK if this isn't described in the comments, or is there a different way to do this?

Reply:
Oh, I see from [the examples here](https://developer.wordpress.org/apis/internationalization/internationalization-guidelines/#variables) that any translatable text doesn't need to be described specifically, only the variables. Makes sense, the suggestion above works 🎉

Although as per @kebbet's [suggestion](https://github.com/WordPress/wordpress-develop/pull/5441#discussion_r1350692864), should we keep the "Hidden accessibility text" comment?